controllers/apply-next/validators.js

```diff
@@ -92,42 +109,24 @@ const budgetValidator = joi => {
                 return value;
             }
         },
-        pre(value, state, options) {
```

Comment:
The ` if (!params.number) {` check isn't needed and the main total budget check can be done directly in the `validate` function. Because the max budget parameter is an explicit value (must be a number) we don't need to do this check ourselves and can get it straight from params.

I think the `_flags` example in the docs was to cover the example of `round` where that is a flag to _mutate_ a value (i.e. round it) rather than validate against it like we're doing here.